go-controller/pkg/ovn/namespace_test.go

```diff
@@ -17,20 +17,24 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newNamespaceMeta(namespace string) metav1.ObjectMeta {
+func newNamespaceMeta(namespace string, additionalLabels map[string]string) metav1.ObjectMeta {
+	labels := map[string]string{
+		"name": namespace,
+	}
+	for k, v := range additionalLabels {
+		labels[k] = v
+	}
 	return metav1.ObjectMeta{
-		UID:  types.UID(namespace),
-		Name: namespace,
-		Labels: map[string]string{
-			"name": namespace,
-		},
+		UID:         types.UID(namespace),
+		Name:        namespace,
+		Labels:      labels,
 		Annotations: map[string]string{},
 	}
 }
 
-func newNamespace(namespace string) *v1.Namespace {
+func newNamespace(namespace string, additionalLabels map[string]string) *v1.Namespace {
```

Comment:
rather than adding a new argument which 90% of callers are going to pass `nil` for, it would be cleaner to add a new `newNamespaceWithLabels`